Automattic/woocommerce-payments
# Use a singular label in the summary of Transactions and Deposits lists

Use the _n method to render the proper label depending on the number

Fixes #2107 

#### Changes proposed in this Pull Request
The Transactions and Deposits list summary were using a hardcoded label no matter what number they were referring to.

The change introduces the usage of `_n` to display a different label for singular and plural.



Before:
![image](https://user-images.githubusercontent.com/2612426/125778857-76147896-5a04-4d1d-a553-da61af23d0e0.png)

![image](https://user-images.githubusercontent.com/2612426/125779074-7df9d247-f9ef-4402-8e2d-2ecba2d37721.png)

After:
![image](https://user-images.githubusercontent.com/2612426/125778958-39239060-9770-4953-8fb0-feb0b43e8722.png)

![image](https://user-images.githubusercontent.com/2612426/125779003-b5063498-57a8-420a-934b-7cbc55d02ccb.png)

#### Testing instructions





* Load only a transaction/deposit
* See the right label (as singular)
* Increase the count
* See the right label (as plural)

-------------------
